### PR TITLE
refactor: refactor hooks useEventBroker and useLayer

### PR DESCRIPTION
**Pull request by NikitaCG**
Hook `useEventBroker` and `useLayer` very connected with their classes, therefore, there was no point in taking them out separately. Instead, I refactored folder for these utilities

### CODE REVIEW

**Review by korvin89 — COMMENTED** [review]
About naming: we are trying to use kebab-case for folders and files except react component  files naming that uses camel case. Could u rename `utils/eventBroker` -> `utils/event-broker`?

**Comment by NikitaCG** [comment]
> About naming: we are trying to use kebab-case for folders and files except react component files naming that uses camel case. Could u rename `utils/eventBroker` -> `utils/event-broker`?

done

**Review by korvin89 — APPROVED** [review]

File: src/components/utils/layer-manager/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export {layerManager, getLayersCount} from './LayerManager';
+export type {
+    LayerConfig,
+    LayerExtendableProps,
+    ContentElement,
```
Comment (amje) [inline src/components/utils/layer-manager/index.ts:5]: Not sure if we want LayerExtendableProps and ContentElement to be exposed

**Review by amje — CHANGES_REQUESTED** [review]

File: src/components/utils/layer-manager/index.ts (hunk 1 above)
Comment (NikitaCG) [reply src/components/utils/layer-manager/index.ts]: done

**Review by amje — APPROVED** [review]

**Pull request merged**